contracts/dapis/DapiServer.sol

```diff
@@ -468,24 +460,21 @@ contract DapiServer is
         bytes32 subscriptionId, // solhint-disable-line no-unused-vars
         bytes calldata data,
         bytes calldata conditionParameters
-    ) external override returns (bool) {
-        require(msg.sender == address(0), "Sender not zero address");
```

Comment:
These checks ensure that the function can only be called by a [VoidSigner](https://docs.ethers.io/v5/api/signer/#VoidSigner) because that's the only way `msg.sender` can be `address(0)`. Since VoidSigners can only do static calls, this also means these functions can only be static-called. There are two caveats with this (that we didn't encounter yet): (1) Node implementations of some chains may not support VoidSigner calls (2) `address(0)` can be used to send transactions to implement chain-specific functionality such as roll-ups. So I was never 100% comfortable about this but at least ToB thought it was fine.

The entire reason why we're trying to limit calls to static calls is to prevent people working around whitelisting. For example, if we exposed this contract, other contracts would be able to extract 1 bit of information from the data feed (through the returned `bool`). Since we don't need to worry about this anymore, I removed all those `require`s.